code/game/machinery/computer/camera.dm

```diff
@@ -201,23 +201,17 @@
 
 // Returns the list of cameras accessible from this computer
 /obj/machinery/computer/security/proc/get_available_cameras()
-	var/list/L = list()
-	for (var/obj/machinery/camera/C in GLOB.cameranet.cameras)
+	var/list/camlist = list()
+	for(var/obj/machinery/camera/cam in GLOB.cameranet.cameras)
```

Comment:
```suggestion
	for(var/obj/machinery/camera/cam as() in GLOB.cameranet.cameras)
```

Reply:
are we sure the camera is not null, because it could be qdeleted